app/controllers/saml_idp_controller.rb

```diff
@@ -146,7 +146,7 @@ def log_external_saml_auth_request
     analytics.saml_auth_request(
       requested_ial: requested_ial,
       authn_context: saml_request&.requested_authn_contexts,
-      requested_aal_authn_context: saml_request&.requested_aal_authn_context,
+      requested_aal_authn_context: FederatedProtocols::Saml.new(saml_request).aal,
```

Comment:
hm, i guess this is conceivably a place where we'd want it to be `nil`?

Reply:
That's what I thought and why I left it alone in `saml_idp_auth_concern`.